doc/lua_api.txt

```diff
@@ -2201,13 +2201,17 @@ appearing when you don't expect them to. See `no_prepend[]`
 
 #### `textarea[<X>,<Y>;<W>,<H>;<name>;<label>;<default>]`
 * Same as fields above, but with multi-line input
-* if the text overflows a vertical scrollbar is added
-* if the name is empty the textarea is readonly, the label is not displayed.
+* If the text overflows, a vertical scrollbar is added.
+* If the name is empty, the textarea is readonly and
```

Comment:
'read only'?

Reply:
I agree, it should be changed to "read-only".